data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+The "recommended" image will be updated on a monthly basis during DP0.1 to encourage users to adapt to using software that is an active development, and to benefit from the bug fixes and updates made by Rubin staff.
+Older images will remain accessible to users.
+
+RSP users who are doing a lot of image processing might need to select a large server, and those who are working with small subsets of catalog data can use a small server.
+
+**Start the Server:**
+Pressing the orange start button to start the server returns this page with a blue progress bar.
+
+.. figure:: /_static/RSP_NB_progress_bar.png
+    :width: 400
+    :name: RSP_NB_progress_bar
+
+    Be patient. Sometimes it takes a couple of minutes to start a server.
+
+**JupyterLab Navigation:**
+The JupyterLab landing page in the figure below is the launch pad for all JupyterLab functionality (e.g., Notebook, Terminal).
+Return to this launch pad at any time by clicking the plus symbol at upper left.
+For DP0.1, only Python Notebooks and the terminal interface are supported (i.e., not the Console), and users are not able to access Portal queries from a Jupyter Notebook.
+
+.. figure:: /_static/RSP_NB_launcher_options.png
+    :width: 400
+    :name: RSP_NB_launcher_options
+
+    The JupyterLab landing page from which several tools can be launched and the file system can be browsed (left sidebar).
+
+In the very left-most vertical sidebar of icons, the top icon is a file folder, and that is the default view.
+The left sidebar lists folders in the user's home directory (e.g., DATA, WORK, and notebooks).
+Launching a terminal (the default is a linux bash terminal) and using the command "ls" will return the same list.
+Navigate the file system and open files by double-clicking on folders and files in the left sidebar.
+All users will find a set of tutorial notebooks provided in the notebooks directory (:ref:`Examples-DP0-1-Notebooks`).
+Jupyter Notebooks can be identified by their suffix ".ipynb". 
+
+**Using a Terminal:**
+The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal. As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute "source ${LOADSTACK}" and "setup lsst_distrib". For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_. Type "butler --help" in any terminal to see a list of available butler functionality.
+
+**Safely Log Out of JupyterLab:**
+Use the File item in the top menu bar.
+To safely shut down a Notebook, choose Close and Shutdown Notebook.
+To safely shut down a JupyterLab server and log out of the RSP, choose Save all, Exit, and Log Out.
```

Comment:
It would be good to call this out even louder.  We plan on culling users idle pods, which means they may have to log in again the next time they come around anyway.  If we can train them to just log out when they are done, it will make it less jarring since they will expect to have to log back in anyway.